File: cpp/driver.cpp
#include "data_cleaner_manual.h"
#include "log_reg.h"
#include <time.h>

int main(){
	clock_t start, middle, end;
	start = clock();
	data_cleaner_manual();
	middle = clock();
	log_reg();
	end = clock();
	printf("total runtime = %f\n", ((double)(end-start))/CLOCKS_PER_SEC);
	printf("data cleaner runtime = %f\n", ((double)(middle-start))/CLOCKS_PER_SEC);
	printf("log reg runtime = %f\n", ((double)(end-middle))/CLOCKS_PER_SEC);
	return EXIT_SUCCESS;
}

File: java/Driver.java
import java.io.*;
import model.LogReg;
import model.DataCleanerManual;

class Driver{
	public static void main(String[] args) throws IOException {
		double start, end, intermediate;
		start = System.currentTimeMillis();
		DataCleanerManual.data_cleaner_manual();
		intermediate = System.currentTimeMillis();
		LogReg.log_reg();
		end = System.currentTimeMillis();;
		System.out.println("total runtime = "+String.valueOf((end-start)/1000));
		System.out.println("data cleaner runtime = "+String.valueOf((intermediate-start)/1000));
		System.out.println("log reg runtime = "+String.valueOf((end-intermediate)/1000));
		return;
	}
}
